libs/api/domains/rights-portal/src/lib/payment/dto/paymentOverview.input.ts

```diff
@@ -2,11 +2,11 @@ import { Field, InputType, Int } from '@nestjs/graphql'
 
 @InputType('RightsPortalPaymentOverviewInput')
 export class PaymentOverviewInput {
-  @Field(() => String)
-  dateFrom!: string
+  @Field(() => Date)
+  dateFrom!: Date
 
-  @Field(() => String)
-  dateTo!: string
+  @Field(() => Date)
+  dateTo!: Date
 
   @Field(() => String)
   serviceTypeCode!: string
```

Comment:
Won't this break gql queries which provide stringified date inputs?

See [here](https://github.com/island-is/island.is/blob/main/libs/service-portal/health/src/screens/Payments/PaymentOverview.tsx)